store/tikv/gcworker/gc_worker.go

```diff
@@ -283,6 +324,22 @@ func (w *GCWorker) getOracleTime() (time.Time, error) {
 	return time.Unix(sec, nsec), nil
 }
 
+func (w *GCWorker) checkGCEnable() (bool, error) {
+	str, err := w.loadValueFromSysTable(gcEnableKey)
```

Comment:
The SQL should be "select gcEnableKey for update". Otherwise this txn cannot detect others modify the gcEnableKey during the transaction.

Reply:
The sql in `loadValueFromSysTable` is already `select XXX for update`

Reply:
Got